lib/apimatic-core/utilities/signature_verifier_helper.rb

```diff
@@ -0,0 +1,81 @@
+module CoreLibrary
+  # === SignatureVerifierHelper ===
+  # Utility class for extracting headers and body from Rack::Request (or request-like objects).
+  #
+  # All methods are class methods and can be used without instantiating.
+  class SignatureVerifierHelper
```

Comment:
This class should be named as RackRequestHelper as it contains utilities specific to RackRequest.